src/rollover.py

```diff
@@ -246,10 +247,8 @@ def wait_for_all_services(ecs_client, services_on_instance, service_events):
         service_events[service_id].append(event)
 
         if not completed:
-            service = service_descriptions[service_id]
-            print "ERROR: Timeout while waiting for %s to reach steady state" % (service['serviceName'])
-            return False
```

Comment:
It looks like previously this would stop as soon as one service failed to reach steady state, while now it continues trying the other services. Just wanted to check this is an intended behavior change?